data/src/languages_update.py

```diff
@@ -35,92 +39,63 @@ def _detect_best_script_name(
         #   https://www.unicode.org/Public/13.0.0/ucd/Scripts.txt
         return script_probs[0][0], script_probs[0][1]
 
+def _get_alias(value: str) -> str:
+    """Takes a script ID string from _detect_best_script_name()
+    and returns the ISO 15924 code for that script.
 
-def _remove_duplicates(script_dict: Dict[str, str]) -> Dict[str, str]:
-    '''
-    If a values in lang["script"] appears more than once, the [key:value] pair that does not conform to ISO unicode
-    entries returned from unicodedataplus.property_value_aliases['script'] is deleted.
-
-    eg)
-
-    "aze": {
-        "iso639_name": "Azerbaijani",
-        "wiktionary_name": "Azerbaijani",
-        "wiktionary_code": "az",
-        "casefold": true,
-        "script": {
-            "latn": "Latin",
-            "cyrl": "Cyrillic",
-            "ara": "Arabic",
-            "arab": "Arabic"
-        }
-
-        ->
-
-        "aze": {
-        "iso639_name": "Azerbaijani",
-        "wiktionary_name": "Azerbaijani",
-        "wiktionary_code": "az",
-        "casefold": true,
-        "script": {
-            "latn": "Latin",
-            "cyrl": "Cyrillic",
-            "arab": "Arabic"
-        }
-
+    Example: "Arabic" -> "arab"
+    """
+    return ''.join(unicodedataplus.property_value_aliases['script'][value]).lower()
 
-    '''
+def _remove_mismatch_ids(script_dict: Dict[str, str]) -> Dict[str, str]:
+    """If a values in lang["script"] appears more than once, the [key:value] pair that does not conform to ISO unicode
```

Comment:
Can you make the documentation respect the 80 characters in line limit please?

Reply:
yup! I'm going to run `black -79` on everything before I leave today.